.github/workflows/run-tests.yml

```
@@ -0,0 +1,72 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-16.04
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        php: [5.4, 8.0, 7.4, 7.3, 7.2, 7.1, 7.0, 5.6, 5.5]
+        imagemagic: [
+          7.0.10-27,
+          7.0.8-4,
+          7.0.1-0,
+          git7,
+          6.9.2-0,
+          6.8.7-0,
+          6.7.8-0,
+          git6
+        ]
+        exclude:
+          - php: 5.4
+            imagemagic: 6.8.7-0
```

I Think expanding this to other failing cases (PHP8 on older imagemagick) might be an idea?

```suggestion
        exclude:
          - php: 5.4
            imagemagic: 6.8.7-0
          - php: 5.6
            imagemagic: 7.0.1-0
          - php: 7.1
            imagemagic: 6.7.8-0
          - php: 7.2
            imagemagic: 6.7.8-0
          - php: 7.3
            imagemagic: 6.7.8-0
          - php: 7.4
            imagemagic: 6.7.8-0
          - php: 8.0
            imagemagic: 6.7.8-0
```

It looks like 6.7.8-0 was spotty at best (broke and then fixed on some PHP, but then back to broken). Maybe it's the `.phpt` that needs fixing. IDK, it feels like maybe it's fine to let it go as it's an old lib, not a new one?

---

I'll leave that choice up to @Danack